.eslintrc.js

```diff
@@ -146,10 +146,10 @@ module.exports = {
           'Using default React import is discouraged, please use named exports directly instead.',
       },
       {
-        // forbid <a> in favor of <LinkButton> or <ExternalLink>
+        // forbid <a> in favor of <Button type={'link'}> or <ExternalLink>
         selector: 'JSXOpeningElement[name.name="a"]',
         message:
-          'Using <a> is discouraged, please use <LinkButton> or <ExternalLink> instead.',
+          'Using <a> is discouraged, please use <Button type={link}> or <ExternalLink> instead.',
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
          'Using <a> is discouraged, please use <Button type="link"> or <ExternalLink> instead.',
```